src/components/EditCoursePage/CollapsibleCourseRunFields.jsx

```diff
@@ -0,0 +1,208 @@
+            {'View about page'}
+          </a>
+        </div>
+      </div>
+    );
+  }
+  return (
+    <div>
+      <span>Your new course run</span>
+    </div>
+  );
+};
+
+class CollapsibleCourseRunFields extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleRemove = this.handleRemove.bind(this);
+  }
+
+  handleRemove(index) {
+    this.props.fields.remove(index);
+  }
+
+  render() {
+    const { fields, courseRuns, languageOptions } = this.props;
+    return (
+      <div>
+        {fields.map((courseRun, index) => (
+          <Collapsible
+            title={formatCourseRunTitle(courseRuns[index])}
+          >
+            <Field
+              name={`${courseRun}.start`}
+              type="date"
+              component={RenderInputTextField}
+              format={value => moment(value).format('YYYY-MM-DD')}
+              label={
+                <React.Fragment>
+                  Start date
+                  <span className="required">*</span>
+                </React.Fragment>
+              }
+              required
+            />
+            <Field
+              name={`${courseRun}.end`}
+              type="date"
+              component={RenderInputTextField}
+              format={value => moment(value).format('YYYY-MM-DD')}
+              label={
+                <React.Fragment>
+                  End date
+                  <span className="required">*</span>
+                </React.Fragment>
+              }
+              required
+            />
+            <Field
+              name={`${courseRun}.pacing_type`}
+              type="text"
+              component={RenderSelectField}
+              options={
+                [
+                  {
+                    label: 'Instructor-paced',
+                    value: 'instructor_paced',
+                  },
+                  {
+                    label: 'Self-paced',
+                    value: 'self_paced',
+                  },
+                ]
+              }
+              label={
+                <React.Fragment>
+                  Course Pacing
+                </React.Fragment>
+              }
+            />
+            <Field
+              name={`${courseRun}.min_effort`}
+              type="number"
+              component={RenderInputTextField}
+              label={
+                <React.Fragment>
+                  Minimum Effort
+                </React.Fragment>
+              }
+            />
+            <Field
+              name={`${courseRun}.max_effort`}
+              type="number"
+              component={RenderInputTextField}
+              label={
+                <React.Fragment>
+                  Maximum Effort
+                </React.Fragment>
+              }
+            />
+            <Field
+              name={`${courseRun}.pacing_type`}
+              type="text"
+              component={RenderSelectField}
+              options={
+                [
```

Comment:
Never mind, does seem worthwhile to move this to a variable or function since it's used in multiple places